gapic.sh

```diff
@@ -85,10 +85,19 @@ if [ "$(ls -A $OUT )"]; then
   >&2 echo "Warning: Output directory is not empty."
 fi
 
+# Convert IN and OUT to absolute paths for Docker
+CWD=`pwd`
+cd ${PROTO_PATH}/$IN
+ABS_IN=`pwd`
+cd $CWD
+cd $OUT
+ABS_OUT=`pwd`
+cd $CWD
```

Comment:
Outrageous nit: why not do
```bash
ABS_IN=$(realpath ${PROTO_PATH}/${IN})
ABS_OUT=$(realpath ${OUT})
```
?

Reply:
There is no `realpath` (or `readlink -f`) on OS X (at least by default). The current script runs without any problems on OS X and I want to keep it that way.

Reply:
Aha! Good to know.